common/src/main/java/com/wynntils/features/ui/CustomSeaskipperMapScreenFeature.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.ui;
+
+import com.wynntils.core.config.Category;
+import com.wynntils.core.config.ConfigCategory;
+import com.wynntils.core.features.Feature;
+import com.wynntils.mc.event.ScreenOpenedEvent;
+import com.wynntils.screens.maps.SeaskipperMapScreen;
+import com.wynntils.utils.mc.ComponentUtils;
+import com.wynntils.utils.mc.McUtils;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@ConfigCategory(Category.UI)
+public class CustomSeaskipperMapScreenFeature extends Feature {
```

Comment:
I think `CustomSeaskipperScreenFeature` is enough. Dont forget to rename the i18n as well.